env/includes/settings/apps/tk-multi-publish2.yml

```diff
@@ -42,6 +42,11 @@ settings.tk-multi-publish2.ingest.create_version:   '@settings.tk-multi-publish2
 settings.tk-multi-publish2.ingest.upload_version:   '@settings.tk-multi-publish2.basic.upload_version'
 
 
+settings.tk-multi-publish2.library.collector:        '@settings.tk-multi-publish2.ingest.collector:{config}/tk-multi-publish2/library/collector.py'
+settings.tk-multi-publish2.library.publish_files:    '@settings.tk-multi-publish2.ingest.publish_files:{config}/tk-multi-publish2/library/library_files.py'
```

Comment:
"library" is not an action. I would probably just call this "ingest_files.py" and use the parent folder to discriminate it from regular ingest.